### PR TITLE
1.7.0-beta7 release

### PR DESCRIPTION
**Pull request by outofambit**
- [x] finalize release notes
- [x] release to beta

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "1.7.0-beta7": [
+      "[Improved] Improve error message when publishing repo to an organization - #7472",
+      "[Improved] \"View Stash\" button accessibility improvements - #7274",
+      "[Improved] Performance improvements for bringing changes to another branch - #7471",
+      "[Improved] Perofmance improvements for detecting conflicts from restoring a stash - #7476"
```
Comment (shiftkey) [inline changelog.json]:
```suggestion
      "[Improved] Performance improvements for detecting conflicts from a restored stash - #7476"
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "1.7.0-beta7": [
+      "[Improved] Improve error message when publishing repo to an organization - #7472",
```
Comment (shiftkey) [inline changelog.json]:
```suggestion
      "[Improved] Error message when unable to publish private repository to an organization - #7472",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "1.7.0-beta7": [
+      "[Improved] Error message when unable to publish private repository to an organization - #7472",
+      "[Improved] \"View Stash\" button accessibility improvements - #7274",
```
Comment (billygriffin) [inline changelog.json]: Isn't this the button in the changes panel?
```suggestion
      "[Improved] \"Stashed changes\" button accessibility improvements - #7274",
```

Comment (outofambit) [reply changelog.json]: ah yes good point

**Comment by outofambit** [comment]
this 🚢-ed!

**Pull request merged**